### ISSUE TITLE
[Routing] Trailing Slash article should be changed for 4.1

### ISSUE THREAD

**Issue by vudaltsov** [symfony/symfony-docs#9761]
It looks like since `4.1` and symfony/symfony#26283 article https://symfony.com/doc/current/routing/redirect_trailing_slash.html is no longer needed.

Besides implementing such a controller in `4.1` will cause infinite redirect from no trailing slash to trailing slash and back for non matching urls.

**Comment by javiereguiluz** [comment]
You are right! However, this change is already done. In 4.1 this article no longer exists: https://symfony.com/doc/4.1/routing/redirect_trailing_slash.html  Closing then as fixed. Thanks!

**Comment by xabbuh** [comment]
For the records, the changes was made in #9385.

**Comment by vudaltsov** [comment]
I think it would be nice to attract community's attention to this somehow. Recently I updated I've of my projects to 4.1 beta for test and for about an hour couldn't understand why I had infinite redirects on favicon.ico (which didn't exist).

Only after thorough analysis I realized I had to remove my redirect controller from routes.yaml.
